storey/utils.py

```diff
@@ -105,4 +105,19 @@ def _split_path(path):
 def url_to_file_system(url):
     parsed_url = urlparse(url)
     schema = parsed_url.scheme.lower()
+    load_dependencies(schema)
     return fsspec.filesystem(schema), parsed_url.path
+
+
+def load_dependencies(schema):
```

Comment:
Generic name, but actually this is only for s3.

Reply:
will be used for other fs in the future, like azure

Reply:
Probably would've been better named as `load_filesystem_dependencies`  or `load_fs_dependencies `